_meeting-notes/06-08~01-10-2019.md

```diff
@@ -0,0 +1,508 @@
+# Test Ground Weeklys Aug 06 ~ Oct 01 2019
+
+Originally at https://docs.google.com/document/d/1yYTXLOd-Rn9U-yfngaNJAUhiVRiZV6Rr555E4DWTYcQ/edit#
+
+## Resources
+
+- [Trello Board](https://trello.com/b/O9z3ljaH/testground-ignition-%F0%9F%9A%80)
+- [Testing Infra Design](https://github.com/ipfs/testground/blob/master/docs/SPEC.md)
+- [Testing Inventory](https://github.com/ipfs/testground/blob/master/docs/test-inventory.md)
+- [Testing Improvement Tracker](https://docs.google.com/spreadsheets/d/1xyqyGUF-oe3x9ln88YonVeOMWWdknik74lVgL_3dBY8/edit#gid=0)
+
+## Oct 01
+
+### Agenda
+
+- **Updates**
+    - General update from the Cross Team Meeting
+    - Raul
+       - Done
+          -
+       - Next
+          - All of the architecture is documented
```

Comment:
All is a tad too general. @raulk can you list the trello cards for this task here?